implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -310,9 +313,12 @@ private void send(List<String> metricLines) {
                     .withPlainText(body)
                     .send()
                     .onSuccess(response -> handleSuccess(metricLines.size(), response))
-                    .onError(response -> logger.error("Failed metric ingestion: Error Code={}, Response Body={}", response.code(), response.body()));
+                    .onError(response -> logger.error("Failed metric ingestion: Error Code={}, Response Body={}",
+                        response.code(),
+                        StringUtils.truncate(response.body(), LOG_RESPONSE_BODY_TRUNCATION_LIMIT)));
```

Comment:
What do you think about signaling that the body was truncated?
E.g.: if the length of `response.body()` is more than the limit, add `"[...]"` or `(truncated)`.

Reply:
I think it would make sense so that it is reusable across the code base.

Reply:
Added `StringUtils.truncate` + tests for it and changed the exporter to use it.